client/app/authenticators/zap-api-authenticator.js

```diff
@@ -17,15 +17,12 @@ export default class ZAPAuthenticator extends OAuth2ImplicitGrantAuthenticator {
     }
 
     let attemptedLoginEmail = '';
+    const NYCIDUser = jwtDecode(accessToken);
 
     // attempt to decode the token to include email address for debugging
-    try {
-      const { mail } = jwtDecode(accessToken);
+    const { mail } = NYCIDUser;
```

Comment:
Why remove the try/catch?

Reply:
i had it there because it was for debugging purposes and didn't want it to break anything. now we need it